apk-generator/v2/app/static/js/main.js

```diff
@@ -225,6 +233,7 @@ $form.submit(function (e) {
     data.append("email", $emailInput.val());
     data.append("data-source", dataSourceType);
     data.append("build-type", buildType);
+    data.append("is-auth-enabled", isAuthEnabled);
```

Comment:
I did not know the exact command for this. `$authOptionCheckbox.is(':checked'); `
https://stackoverflow.com/questions/2204250/check-if-checkbox-is-checked-with-jquery